MetaMask Plugin/How to create a MetaMask Plugin

```diff
@@ -0,0 +1,26 @@
+Introduction
+
+1. Metamask plugin what’s new? 
+2. How this will improve or advantages over the extension.
+
+
+3.1 BreakDown of new Plugin APIs of Metamask. (Short Video/ Writeup)
+3.2 Explain Approach, and suggest concepts. 
+
+
+4. How to Switch from Extension to Plugin. 
+
+POC
+
+5. POC Demo (Walkthrough)
+5.1. Abstract level code description. 
+5.2. Plugin APIs involved.
+5.3. Implementation
+
+
+Ancillary Material 
+- Metamask Plugin Team 
```

Comment:
Not needed, this is a technical post, though you could mention the governance they discuss in the announcement post

Reply:
Okay, Got it!